plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -370,4 +370,30 @@ module.exports = class ServiceTree extends Tree {
       return serviceTreeLabels;
     }, []);
   }
+
+  getRolesLength() {
+    const name = this.getName();
+
+    return this.reduceItems(
+      function(roles, item) {
+        if (
+          (item instanceof Service || item instanceof Pod) &&
+          item.getRole() !== null
```

Comment:
I think we want to could services even if role is missing, to get an accurate total service count. I think it *should* always have a role, but if its missing its still a service under the group and it should count towards the quota limit status.